package.json

```diff
@@ -31,20 +30,21 @@
   "devDependencies": {
     "@fortawesome/fontawesome-svg-core": "6.1.1",
     "@fortawesome/free-brands-svg-icons": "6.1.1",
+    "@fortawesome/pro-duotone-svg-icons": "6.1.1",
     "@fortawesome/pro-light-svg-icons": "6.1.1",
     "@fortawesome/pro-regular-svg-icons": "6.1.1",
     "@fortawesome/pro-solid-svg-icons": "6.1.1",
-    "@fortawesome/pro-duotone-svg-icons": "6.1.1",
     "@fortawesome/vue-fontawesome": "2.0.6",
     "@storybook/addon-controls": "^6.5.13",
     "@storybook/addon-docs": "^6.5.13",
     "@storybook/addon-storysource": "^6.5.13",
     "@storybook/addon-viewport": "^6.5.13",
     "@storybook/vue": "^6.5.13",
+    "@types/jest": "29.2.0",
```

Comment:
Fix for https://github.com/bethinkpl/design-system/actions/runs/3343339528/jobs/5536454524